Apps/W1/EDocumentsTietoevryConnector/app/src/Tietoevry/Cod96360.TietoevryConnectionEvents.al

```diff
@@ -0,0 +1,113 @@
+namespace TietoevryConnector.TietoevryConnector;
+
+using Microsoft.Sales.Peppol;
+using Microsoft.Sales.Document;
+using Microsoft.Foundation.Company;
+using Microsoft.Sales.Customer;
+using Microsoft.EServices.EDocumentConnector;
+using Microsoft.eServices.EDocument;
+using System.Utilities;
+
+codeunit 96360 "Tietoevry Connection Events"
+{
+    SingleInstance = true;
+    EventSubscriberInstance = StaticAutomatic;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"PEPPOL Validation", OnCheckSalesDocumentOnBeforeCheckCompanyVATRegNo, '', false, false)]
+    local procedure "PEPPOL Validation_OnCheckSalesDocumentOnBeforeCheckCompanyVATRegNo"(SalesHeader: Record "Sales Header"; CompanyInformation: Record "Company Information"; var IsHandled: Boolean)
+    var
+        ExternalConnectionSetup: Record "Tietoevry Connection Setup";
+    begin
+        if not ExternalConnectionSetup.Get() then
+            Error(MissingSetupErr);
+        IsHandled := true;
+        if CompanyInformation."VAT Registration No." = '' then
+            Error(MissingCompInfVATRegNoErr, CompanyInformation.TableCaption());
```

Comment:
Shouldn't those two event subscribers for the PEPPOL Validation be bind manually only for this connector?

Reply:
You tell me, your Pagero-implementation is not bind manually.. code is based on your Pagero connector..

Reply:
How do you propose the e-document framework should handle this?